README.md

```diff
@@ -1,2 +1,42 @@
-# LeaderBoard
-Leaderboard applicaton | JavaScript
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard
+
+> Leader board  application using Js/webpack
+
+
+## Built With
+
+- Js
+- web pack
+
+## Live Demo 
+> 
+
+
+## Getting Started
+This is a simple html/css and js  project.
+
```

Comment:
- Please, the getting started should contain some instructions on how users can have your project run locally on their machine. please add some instruction on how to clone, prerequisites needed to run your app, etc.